src/helpers/month.helper.ts

```diff
@@ -81,8 +71,9 @@ export class MonthHelper {
     const firstMonthDay = new Date(year, month, 1).getDay();
     const lastMonthDay = new Date(year, month + 1, 0).getDay();
     return {
-      daysMissingFromPrevMonth: firstMonthDay === 0 ? 6 : firstMonthDay - 1,
-      daysMissingFromNextMonth: lastMonthDay === 0 ? 0 : 7 - lastMonthDay,
+      daysMissingFromPrevMonth:
```

Comment:
Kurde zrobiłem bardzo podobnie na początku, ale nie działało mi dla, gdy `firstMonthDay` był równy 0 i było dzielenie liczby ujemniej. Nie wpadłem, żeby dodać NUMBER_OF_DAYS_IN_WEEK. Super pomysł :D 
@Qwebeck zerknij sobie, bo razem to kmniliśmy

Reply:
Super wygląda :) 
Jedyne co, to w 
`daysMissingFromNextMonth: (NUMBER_OF_DAYS_IN_WEEK - lastMonthDay) % NUMBER_OF_DAYS_IN_WEEK,`

moduło chyba jest zbędne.

Reply:
@Qwebeck  lastMonthDay może być 0

Reply:
Tak, racja